addon/ng2/blueprints/ng2/files/__path__/index.html

```diff
@@ -22,9 +21,15 @@
   <link rel="apple-touch-icon" sizes="180x180" href="/icons/apple-touch-icon-180x180.png">
   <link rel="apple-touch-startup-image" href="/icons/apple-touch-icon-180x180.png">
   <% } %>
-  
+  <% if (!arguments[0].files) { 
```

Comment:
I believe with our config and logic of injection styles we cannot use `HtmlWebpackPlugin`. Otherwise this line is used because template is going through the lodash `_.template` twice.. first when generating blueprint and then second when generating `index.html` for the `dist/`.